src/main/java/frc/robot/commands/TeleopSwerve.java

```diff
@@ -0,0 +1,46 @@
+package frc.robot.commands;
+
+import frc.robot.Constants;
+import frc.robot.subsystems.Swerve;
+
+import java.util.function.BooleanSupplier;
+import java.util.function.DoubleSupplier;
+
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+
+public class TeleopSwerve extends CommandBase {    
```

Comment:
Recommend ending all command classes with `Command`

Reply:
That is Stealth's standard. Fixed.